site/_includes/macros/render-type.njk

```diff
@@ -506,7 +506,7 @@
     {% elif feature.unknownVersion %}
       {# This is stable but not yet visible in the version history data. It's probably coming next
         Chrome release. #}
-      <span class="aside--warning tag-pill type--label rounded-lg">
+      <span class="aside--warning tag-pill type--label rounded-lg" title="No version data available">
```

Comment:
You might want to use i18n filter here to have this string be translatable.

Reply:
This is normally a good suggestion but our reference documentation has no plan to be translated, since the raw source in Chrome is far away and English-only.

Regardless it's weird to just fix it for a single string in a world where there's lots of them :)